content/post/online_hackathon_an_experiment.md

```diff
@@ -0,0 +1,33 @@
+description = "Experience from my first online Hackathon"
+disableComments = false
+author = "Akshat Pande"
+tags = ['Hackathon','Coding','Bridgingo','First','Python']
+categories = ['Hackathon','Competition']
++++
+
+The first image which comes to a beginner's mind on hearing the word [Hackathon](https://en.wikipedia.org/wiki/Hackathon) is a group of people with professional ethical hacking skills who are racing towards a specific goal to win in a competition, even I thought this in a similar manner until my fellows from [ALiAS](https://asetalias.in) helped me to sort it out.
+
+In layman terms, Hackathon is just a coding competition where people in groups or individual work upon building something out of, either a list of projects/problem-statements or a project based on some theme with the help of mentors with expertise in various fields present at the venue. Also, participants are bound to a time limit usually ranging between 24-52 hours with proper arrangements of basic amenities for living. The same happens in an online mode too where you at your place race to the finish line.
+
+![Hackathon](/blog/img/akshat0047/h1.jpg)
+
+All this started back in mid-February when scrolling across Facebook I glanced upon an advertisement, designed perfectly to ignite a coder's interest into it. It read "Game Of Codes" followed by details of the contest, much appropriate and professional looking that I registered for the same. There were mentions for some fancy top prizes too though I was not expecting myself to grab one. Soon I was informed that the problem statement will be out on 24th February 2019 at 10:30 AM. To be honest I was not serious about this as the hackathon was preceded by an online test involving questions from B.Tech core subjects.
+
+In a moment the day arrived, I still remember it was a prominent day for me not only because of this hackathon but also I had an important performance scheduled early morning at Gomti Nagar, Lucknow. Well, being my passion past few years it was a group dance performance. I was getting frequent reminders from [Bridgingo](https://bridgingo.com/) through emails and as soon as things got wrapped up I gave up for post-performance celebrations to get started with the 1st stage of hackathon i.e. the online test at my place. Geared up with my laptop and knowledge here I sit in 101-A room, Amity University Lucknow moving through questions solving one at a time with best of my efficiency, the email with the test credentials read that the test questions were according to Bloom's Taxonomy which indeed was some list of concepts involving basic IQ skills related to the topic. I do not remember the exact number of questions asked but at least I made more than half of them right and it is the only satisfactory emotion from that time. After the test got over, I was relaxed with mixed feelings making me doubt upon my core knowledge as its a fact once you get hard into coding the classroom subjects seems to be an overhead.
```

Comment:
```suggestion
The day arrived soon, felt like just a moment. I still remember it was an important day for me because, along with the hackathon, I also had a performance early morning at Gomti Nagar, Lucknow. It was a group dance performance, which has been my passion for the last couple of years. I was getting frequent email reminders from [Bridgingo](https://bridgingo.com/). So, as soon as things got wrapped up with the dance, I gave up the post-performance celebrations to get started with the 1st stage of the hackathon. And here I was, geared up with my laptop and knowledge, sitting in room 101-A, Amity University, Lucknow, solving one question at a time. The email with the test credentials also mentioned that the test questions were according to Bloom's Taxonomy. I do not remember the exact number of questions, but I marked more than half of them right; it was the only satisfactory emotion at the time. After the test was over, I was feeling relaxed with some mixed feelings making me doubt my core knowledge as it's a fact that once you get hard into coding, the classroom subjects seem to be an overhead.
```